<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantPage</class>
 <widget class="QWidget" name="AssistantPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 17px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}

QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
  </property>
  <widget class="QLabel" name="label_car3SIVOutputCurrent">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>271</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4SIVOutputCurrent">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>271</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2SIVOutputCurrent">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>271</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_SIVOutputCurrent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>271</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SIV输出电流(A)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2No">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>116</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2SIVOutputVoltage">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>193</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1SIVInputVoltage">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>154</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_SIVInputVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>154</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SIV输入电压(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1SIVOutputRate">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>232</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_vehicleNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>116</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3SIVOutputRate">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>232</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_SIVOutputRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>232</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SIV输出频率(Hz)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4SIVInputVoltage">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>154</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4No">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>116</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1SIVOutputVoltage">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>193</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3SIVOutputVoltage">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>193</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3No">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>116</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_SIVOutputVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>193</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SIV输出电压(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4SIVOutputRate">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>232</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2SIVOutputRate">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>232</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4SIVOutputVoltage">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>193</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1SIVOutputCurrent">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>271</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3SIVInputVoltage">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>154</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1No">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>116</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2SIVInputVoltage">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>154</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_lineVoltage1">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>350</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_extendEnable1">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>310</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_lineVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>母线电压(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2SIVOutputVoltage_2">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>310</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_lineVoltage2">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>350</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2SIVOutputCurrent_2">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>350</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_extendEnable2">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>310</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3SIVOutputCurrent_2">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>350</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3SIVOutputVoltage_2">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>310</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_extendEnable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>扩展使能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset2">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>390</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>429</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut3">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>429</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset1">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>390</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut2">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>429</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut4">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>429</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset4">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>390</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut1">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>429</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset3">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>390</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>98</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cut1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>429</y>
     <width>98</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cut2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>429</y>
     <width>98</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>390</y>
     <width>98</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_EmergencyBroadcast">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>紧急广播</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
